app/api/resources/mentorship_relation.py

```diff
@@ -657,7 +657,14 @@ class UpdateTask(Resource):
     @mentorship_relation_ns.response(
         HTTPStatus.OK, "%s" % messages.TASK_WAS_ACHIEVED_SUCCESSFULLY
     )
-    @mentorship_relation_ns.response(HTTPStatus.BAD_REQUEST, "%s" % messages.TASK_WAS_ALREADY_ACHIEVED)
+    @mentorship_relation_ns.response(
+        HTTPStatus.BAD_REQUEST,
+        "%s\n%s"
```

Comment:
Could you use f-strings, as per these [coding standards](https://github.com/anitab-org/mentorship-backend/wiki/Coding-Standards)

Reply:
I changed it to f-strings!